README.md

```diff
@@ -1529,23 +1532,24 @@ $ helm del --purge tfm-almacar
 
 # 5. Results
 
-TODO
+Summarizing what has been exposed in previous chapters of this document, we should say we have implemented three versions of our application, each one improving its predecessor. As a result after these three iteractions, we have a microservice-based Spring application, implemented to be deployed on a Kubernetes cluster with Helm charts, that could be used as backed for a stock management system.
+The application is able to store items and orders information, with the limitations of its data model described in previous chapters. This information is stored on a MySQL database but other database could be used just adapting one service, thanks to the usage of Spring Data JPA.
+The application offers its functionality via REST API. Although  Kafka and gRPC are used as internal communication mechanisms between our services. Metrics are also published using Prometheus.
 
-We have been able to implement a cloud application using several technologies in a limited amount of time.
 
 # 6. Conclusions and future work
 
-TODO
+Although we have to discard some points we wanted to review at the beggining of the project, we can say we are very satisfied with the work done. We have been able to implement a cloud application using several technologies in a limited amount of time, and taking into account we had no previous experience with them. It is amazing to experiment the huge amount of available tools, and how it is possible to build something from scratch combining some of them to solve your use case of choice. At the same time, all these technologies are alive, all of them have people working to improve them and it is easy to collaborate if you have a proposal or find a bug.
+
+We have also experiment how a microservice project can grow easily, being difficult to run it locally, which creates the opportunity for the PaaS vendors which are offering "Kubernetes as a service". Minikube is enough for small projects, but we had resources problems when trying to adapt Elastic, although we have just one replica per deployment.
+
+Using microservices, it has been easier to divide tasks among us. This is an advantage of this kind of architectures, as different teams can take ownership of different services and work in parallel, speeding the application development.
```

Comment:
I would say in this part that it was great for us becuuse we could practice with technologies we are using right now in the project we are starting.